docs/src/main/sphinx/security/file-system-access-control.md

```diff
@@ -432,7 +432,8 @@ any catalog, and allows all users to create, drop, and execute functions (includ
 (system-file-procedure-rules)=
 #### Procedure rules
 
-These rules control the ability of a user to execute procedures.
+These rules control the ability of a user to execute procedures using the
+`CALL <catalog>.<schema>.<procedure>(<arguments>)` statement.
```

Comment:
Link to docs for it.. 

```suggestion
[CALL <catalog>.<schema>.<procedure>(<arguments>)](/sql/call) statement.
```

Also .. those docs are minimalistic... sigh

Reply:
But they are still more descriptive than the simple example I gave :) I'll just shorten this to `[CALL](/sql/call)` then.